docs/html/user_guide.rst

```diff
@@ -469,20 +478,20 @@ pip's command line options can be set with environment variables using the
 format ``PIP_<UPPER_LONG_NAME>`` . Dashes (``-``) have to be replaced with
 underscores (``_``).
 
-For example, to set the default timeout::
-
-    export PIP_DEFAULT_TIMEOUT=60
-
-This is the same as passing the option to pip directly::
+For example, setting ``PIP_DEFAULT_TIMEOUT`` to 60 has the same effect as
+passing the option to pip directly::
 
     pip --default-timeout=60 [...]
 
-For command line options which can be repeated, use a space to separate
-multiple values. For example::
-
-    export PIP_FIND_LINKS="http://mirror1.example.com http://mirror2.example.com"
+For command line options which can be repeated, if they do not take any argument
+like ``--quite`` or ``--verbose``, use a non-negative integer to represent
+the number of repetition, for instance, setting ``PIP_VERBOSE`` to 3 is
+equivalent to calling pip with ``-vvv``.
```

Comment:
I feel the previous rendering (using code blocks) is more readable than prose form.